### ISSUE TITLE
Dnsapi

### ISSUE THREAD

**Issue by Imperatorn** [etcimon/windows-headers#13]
Do you want to include my (undocumented) dnsapi or do you have plans on adding the yourself?

**Comment by etcimon** [comment]
Yes, it does look like it's not included here yet, I could add it myself when I have time in a week or so, if you want to do it though the types and imports need to be adjusted and pragma lib added
